car_cleaning_service_without_threads.rb

```diff
@@ -0,0 +1,198 @@
+require 'time'
+
+class Car
```

Comment:
you can extract `Car` class intro a separate class since you are reusing it

Reply:
I am refactoring the code to have multiple files, as it is easier to work with cleaner code, so I will extract it and move the classes into packages.